src/Storage/Commands.Storage/Blob/Cmdlet/GetAzureStorageBlobContent.cs

```diff
@@ -26,7 +26,7 @@
 
 namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
 {
-    [Cmdlet(VerbsCommon.Get, StorageNouns.BlobContent, ConfirmImpact = ConfirmImpact.High, DefaultParameterSetName = ManualParameterSet),
+    [Cmdlet(VerbsCommon.Get, StorageNouns.BlobContent, SupportsShouldProcess = true, DefaultParameterSetName = ManualParameterSet),
```

Comment:
What's the default ConfirmImpact?  We would like to have the ConfirmImpact= low for this cmdlet.